### PR TITLE
Pyro cleanup

### PR DESCRIPTION
**Pull request by jandryuk**
This PR cotains two small cleanups after the pyro uprev.  Remove needed variables and ensure we reference pyro in our config going forward.

### CODE REVIEW

File: example-config [hunk 1]
```diff
@@ -35,9 +35,9 @@ XENCLIENT_TAG=oe-upgrade-pyro
 # The commit hash for meta-java is the last commit that had openjdk6 and idedtea6
 META_JAVA_TAG=pyro
 
-# Default to the same branch of meta-selinux as OE_BRANCH (e.g. jethro).
+# Default to the same branch of meta-selinux as OE_BRANCH (e.g. pyro).
 # Uncomment and edit if you need a specific commit from meta-selinux.
-#META_SELINUX_TAG=jethro
+#META_SELINUX_TAG=pyro
```
Comment (eric-ch) [inline example-config]: Unfortunately, [meta-selinux](https://git.yoctoproject.org/cgit/cgit.cgi/meta-selinux/refs/heads) does not have a pyro branch (as of today at least). [OpenXT `master` branch](https://github.com/OpenXT/openxt/blob/master/.gitmodules#L20) therefor tracks the `meta-selinux` `master` branch, `stable-8` is on a fixed known good revision. In this case it seems this should be set to `master` by default... Actually it should but PRs got swapped... Fixing soon.

Comment (jandryuk) [reply example-config]: In the back of my head, I knew I was just changing a commented it like. So while I built with this change, I knew it wasn't actually changing the repo. My apologies for not checking meta-selinux.

Comment (eric-ch) [reply example-config]: Actually, it made me to look in `stable-8` & `master` `.gitmodules` which were not right, so thanks for missing that! :)

**Comment by eric-ch** [comment]
@jandryuk Could you PR against stable-8 as well please?

**Comment by jandryuk** [comment]
For stable-8, you want the commit from the submodule, d855c624f32c5e599bf27e06cb8f5b25b3aae12d ?

Looking at example-config again, for both master and stable-8, there are two entries for META_SELINUX_TAG.  One is active and set to master.  The second one is commented out and set to jethro.  I didn't notice this earlier.  Should the second commented out one just be dropped?  Should stable-8 set the uncommented tag to the above hash?

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1452

**Comment by eric-ch** [comment]
@jandryuk Sorry I never replied here...
> For stable-8

It match the git submodule revisions from stable-8 branch, I have:
```
 eddab52a4e7c49bfcdc2c26004d0bc777a1e5c25 build/repos/bitbake
 42072ef80e0a3dfc65188778b471f0db9264520e build/repos/meta-intel
 0c27b120aa508e4bb41394b8dd3645949a611128 build/repos/meta-java
 dfbdd28d206a74bf264c2f7ee0f7b3e5af587796 build/repos/meta-openembedded
 d855c624f32c5e599bf27e06cb8f5b25b3aae12d build/repos/meta-selinux
 45ad257a1e4a6707c376d2f7eb26c3c8bdf03607 build/repos/meta-virtualization
 5e4b4874c4d81b62a32b3836d1c22ecc4c5479c6 build/repos/openembedded-core
```

> XENCLIENT_TAG=oe-upgrade-pyro

That is a silly commit from my part... Sorry it was the name of the dev branch I was using and I must have pushed it without realizing.

**Comment by jandryuk** [comment]
I'm not particularly familiar with the build system and its configuration.  I tend to get it setup once and then just manually change repo revisions, so I have some questions.

We have two build flavors: build-scripts and do_build.sh.  build-scripts uses git submodules for repo configuration.  do_build.sh uses setup_build-next.sh and .config to checkout repos.  .config usually comes from example-config.

Is this right?

Looking in openxt.git, setup_build is unused and can be removed?

setup_build was updated with the pyro uprev, but I'm thinking example-config should change to be closer to the updates in that.  Outside the scope of this change, but do we want to standardize on a single option for the repos?

**Comment by jandryuk** [comment]
I started working on converting setup_build-next.sh to just use git submodules.  Is there a reason why meta-openxt-{haskell,ocaml}-platform aren't included in the submodules?  Same for xenclient-oe?  It would simplify things to have a unified way of specifying branches.  My approach would use submodules if nothing is defined, but allow the repo and/or tag to override the checked out version.

**Comment by eric-ch** [comment]
> [...]Is this right?

I think so:
- `example-config` is a `.config` example for `do_build.sh` AFAIK.
- `do_build.sh` does not use git submodule, but `setup_build-next.sh` script; which in turn feeds from the `<name-of-the-repo>-{_TAG,_REPO}` variables presumed to be defined in `.config`, with the exception of `XENCLIENT_REPO`/`XENCLIENT_TAG`, `BITBAKE_REPO`/`BB_BRANCH` and `OE_BRANCH` that will be used by default if `<name-of-the-repo>_TAG` is not defined (tedious... also `BRANCH` looks entirely cosmetic which is very misleading...).
- `setup_build` seems to be dead indeed. There is quite a lot of bitrot all around `do_build.sh`...

> Is there a reason why meta-openxt-{haskell,ocaml}-platform aren't included in the submodules

The OpenXT based repos are not included in the submodules, but configured through the `.layer` files (`oe/build.sh` will iterate over `${openxt_layers}`). @jean-edouard correct?

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1474

**Comment by eric-ch** [comment]
@jandryuk Do you have a stable-8 version of this?

Will merge soon.

**Comment by jandryuk** [comment]
I'll make a stable-8 version.  I PR for master first, so it can be reviewed and changed as necessary.  Then once it's finalized, I can backport, cherry-pick or make a new version as necessary for stable-8.

**Comment by eric-ch** [comment]
@jandryuk Ack, thanks again for fixing this!

**Pull request merged**